app/views/expense_items/new.html.erb

```diff
@@ -0,0 +1,35 @@
+
+<nav class="navbar navbar-expand-md navbar-dark bg-bala-blue fixed-top">
+    <div class="container-fluid">
+    <%= link_to category_path(@category.id) do %>
+      <i class="fa-solid fa-arrow-left"></i>
+    <% end %>
+    <a class="navbar-brand auto-center" href="#">Add Expense Item</a>
+    </div>
+</nav>
+<div class="container-fluid position-fixed">
+    <div class="col-md-6 col-sm-12 p-lg-5 w-100 ">
+    <div>
+        <div class="d-flex">
+            <div class="w-100 mt-3">
+              <%= form_with(model: @expenseitem, url: category_expense_items_path, class:"d-flex f-column col-gap-3 ") do |form| %>
+                <div class="field">
+                  <%= form.text_field :name, required: true, placeholder: "Item", class: "w-100 py-1" %>
+                </div>
+                <div class="field">
+                  <%= form.number_field :amount, required: true, placeholder: "Price", class: "w-100 mt-3 py-1" %>
+                </div>
+                <div class="mt-3">
+                      <div class="container-fluid actions bg-green ">
+                      <%= form.submit 'Save', class: "w-100 btn btn-md btn-success bg-green mt-3 mb-3" %>
+                      </div>
+                  </div>
+              <% end %>     
```

Comment:
As mentioned in the requirements, when creating a new expense item, the user needs to fill out the form with the following data: 
- name (mandatory)
- amount (mandatory)
- categories (mandatory at least one)
A field for the last item is missing, kindly add it to the form.